.github/ISSUE_TEMPLATE/release_ticket.md

```diff
@@ -31,14 +31,11 @@ This ticket should be complete by {{ env.vaDueDate }}
 <!-- Define the content for the What's New sections of the app stores here -->
 If there aren't any new features, use standard messaging: We added general improvements and fixed a few bugs.
 
-If there are new features or major changes we want to announce, work with Content to finalize copy.
+If a Flagship team decides to update the App Stores What's New content, their work should be done in a ticket and provided to the release manager, who will then update this section. If not none provide its assumed not to be used. 
```

Comment:
The last sentence sounds like there could be a typo in there? Did you mean something like:

```md
If not provided, it's assumed not to be used.
```

Reply:
Nice catch! You are correct. "If not provided, it's assumed not to be used." for the last sentence.

Reply:
Updated and re-requested Theo's review